src/background.md

```diff
@@ -83,6 +83,11 @@ if you only have 30 of them to review, but as the number grows to 300 or
 
 ## Spaced Repetition
 
+The idea behind _spaced repetition_ is that the brain needs time to process and consolidate 
```

Comment:
The original goal of this section was to introduce the history of spaced repetition. While this paragraph makes sense in isolation, it feels a bit out of place chronologically.

Reply:
the reason i considered adding a paragraph of description about spaced repetition was because there was one for active recall above and here just the history was mentioned, nonetheless i removed it as you felt it was out of place , i'm new to open source so learning through still and thought it would be good to start contributing to learn , sorry i was a nuisance ;-; , will try making better valued contributions from next time